File: checkBalance.py
#How to check an account balance

balance = 325
is_active = True
check_balance = False

if check_balance == True:
  if is_active == True and balance > 0:
    print(f"Your balance is ${balance}.")
  elif is_active == False and balance < 0:
    print("Your account is no longer active")
  elif balance == 0 and is_active == True:
    print("Your account is empty")
  elif balance < 0:
    print("Your balance is negative. Please contact bank")
else:
  print("Thank you. Have a nice day!")

File: check_balance.js
const balance = -325.00;
const checkBalance = true;
const isActive = true;

// code that checks account balance
if (checkBalance === true) {
  if (isActive === true && balance > 0) {
    console.log("Your balance is $" + balance.toFixed(2) + ".");
  } else if (isActive === false) {
   	console.log("Your account is no longer active")
  } else if (balance === 0) {
    console.log("Your account is empty")
  } else if (balance < 0) {
    console.log("Your balance is negative. Please contact bank");
  }
} else {
  console.log("Thank you. Have a nice day!")
}
